noxfile.py

```diff
@@ -274,11 +274,12 @@ def upload_release(session):
             f"Remove dist/ and run 'nox -s build-release -- {version}'"
         )
     # Sanity check: Make sure the files are correctly named.
+    distfile_names = [os.path.basename(f) for f in distribution_files]
```

Comment:
I've been fighting the urge to say this in the last 5 minutes, but would `sorted(map(os.path.basename, distribution_files))` be a bit better?

Reply:
lol, probably. Sometimes it's easy to overlook cases where map is way simpler than a comprehension. Thanks for giving in to your instinct and saying so!